libs/barista-components/copy-to-clipboard/src/copy-to-clipboard.ts

```diff
@@ -89,26 +93,25 @@ export class DtCopyToClipboard implements AfterContentInit, OnDestroy {
   /** Copies the provided content to the clipboard. */
   copyToClipboard(): void {
     if (this._input) {
```

Comment:
Should we also emit the `copyFailed` event when there is no `_input`, i.e. in the else-branch?

Reply:
Yes, thanks for pointing it out.